resources/js/components/Tool.vue

```diff
@@ -66,25 +66,25 @@ export default {
       this.publishing = true;
       Nova.request()
         .post("/nova-vendor/publish/publish")
-        .then((response) => {
+        .then(() => {
           this.error = "";
+          this.updateStatus();
         })
         .catch((error) => {
-          this.error = error.message;
+          this.error = error.response.data.message || error.message;
```

Comment:
Geeft dit niet in bepaalde situaties een uncaught type error voor wanner het error object niet de vorm heeft zoals j misschien verwacht bij `error.response.data.message`

Voorbeeld: https://jsfiddle.net/u5261re7/

Reply:
Ja, klopt, dit gaat mis als `response` niet in `error` zit. (wat het geval is als `error.messge` bestaat)
Je kunt dat fixen door `?.` te gebruiken: `error?.response`.